applications/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def application(request):
+    return render(request, "application.html")
```

Comment:
I know this is just a spike, but something like this could be `TemplateView.as_view(template_name="application.html")` directly in the urls.py (really useful for spikes I find!)

Reply:
Yeah but then I couldn't use a request factory!